### DISCUSSION TITLE
ENABLE_GATSBY_REFRESH_ENDPOINT should be available to set in config.flags

### DISCUSSION THREAD

**Discussion by phil-lgr** [gatsbyjs/gatsby#32413] [Ideas / Feature Requests]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

I think it should be added to https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/src/utils/flags.ts

so we have one place to manage flags

### Reproduction Link

https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/src/utils/flags.ts

### Steps to Reproduce

1. Open gatsby-config.js
2. add ENABLE_GATSBY_REFRESH_ENDPOINT: true, or maybe it should be REFRESH_ENDPOINT: true,

### Expected Result

The refresh endpoint should be enabled via standard Gatsby flags

### Actual Result

We need to set ENABLE_GATSBY_REFRESH_ENDPOINT when running the CLI

### Environment

```shell
System:
    OS: macOS 11.4
    CPU: (8) arm64 Apple M1
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 16.5.0 - /var/folders/z3/1b10ld5s49n70q3n77_6_qch0000gn/T/yarn--1626446434911-0.6160726936976746/node
    Yarn: 1.22.10 - /var/folders/z3/1b10ld5s49n70q3n77_6_qch0000gn/T/yarn--1626446434911-0.6160726936976746/yarn
    npm: 7.19.1 - ~/.nvm/versions/node/v16.5.0/bin/npm
  Languages:
    Python: 2.7.16 - /usr/bin/python
  Browsers:
    Chrome: 91.0.4472.114
    Safari: 14.1.1
```

**Comment by LekoArts** [comment]
Hi!

I don't see any value in adding this to the flags are those are just a mechanism for us to let people test out things that at some point get built-in anyways. Always setting `ENABLE_GATSBY_REFRESH_ENDPOINT` to `true` is not desirable and should be done by the user.

**Reply by phil-lgr** [reply]
As a user I prefer to have all flags and env vars at the same place

**Comment by wardpeet** [comment]
Thank you but we're going to keep it as an env var without adding it to the flags.
